src/utils/hooks.test.ts

```diff
@@ -0,0 +1,55 @@
+import { remove } from '@/helpers/file';
+import { formatCode, detectTerraform } from '@/helpers/terraform';
+
+import { postProcess } from './hooks';
+
+jest.mock('@/helpers/terraform');
+
+describe('postProcess', () => {
+  const projectDir = 'postProcess-test';
+  const generalOptions = { projectName: projectDir, provider: 'aws' };
+
+  describe('given current machine had terraform', () => {
+    beforeAll(async () => {
+      (detectTerraform as jest.Mock).mockImplementation(() => true);
+
+      await postProcess(generalOptions);
+    });
+
+    afterAll(() => {
+      jest.resetAllMocks();
+      remove('/', projectDir);
+    });
+
+    it('runs formatCode', async () => {
+      await expect(formatCode).toHaveBeenCalled();
+    });
+  });
+
+  describe('given current machine did not have terraform', () => {
```

Comment:
```suggestion
  describe('given current machine does not have terraform', () => {
```

(and in the previous block `given current machine has terraform`)

Reply:
By the way: NICE HOOK!!!! ❤️